### ISSUE TITLE
Display hostname that logging into in dnote login

### ISSUE THREAD

**Issue by navado** [dnote/dnote#426]
I'd like to see the URL or alias of the server that I'm logging in.
Currently generic prompt shown regardless the configured endpoint
```
$ dnote login -u someoune@some.where 
  Welcome to Dnote Pro (https://www.getdnote.com).
  [?] password: 
  ✔ logged in
```

**Comment by sungwoncho** [comment]
@navado I agree that it is rather confusing to see the `www.getdnote.com` domain when we are connecting to another domain. Let's address this issue by the next release 0.12.0.

**Comment by sungwoncho** [comment]
Fixed in [v0.11.1](https://github.com/dnote/dnote/releases/tag/cli-v0.11.1). Thanks for the request. Feel free to reopen.
